<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Атаки на стегосистему</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QMdiArea" name="mdiArea"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>950</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_2"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>О программе</string>
    </property>
    <addaction name="action_4"/>
    <addaction name="action_3"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Окна</string>
    </property>
    <addaction name="action_5"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>Инструменты</string>
    </property>
    <widget class="QMenu" name="menu_5">
     <property name="title">
      <string>Фильтры</string>
     </property>
     <addaction name="action_8"/>
    </widget>
    <widget class="QMenu" name="menu_6">
     <property name="title">
      <string>Анализаторы</string>
     </property>
     <widget class="QMenu" name="menuJPEG">
      <property name="title">
       <string>JPEG</string>
      </property>
     </widget>
     <widget class="QMenu" name="menuGIF">
      <property name="title">
       <string>GIF</string>
      </property>
      <addaction name="action_11"/>
     </widget>
     <addaction name="menuJPEG"/>
     <addaction name="menuGIF"/>
     <addaction name="action_9"/>
    </widget>
    <addaction name="menu_5"/>
    <addaction name="menu_6"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_4"/>
   <addaction name="menu_3"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action"/>
   <addaction name="action_9"/>
   <addaction name="action_8"/>
  </widget>
  <action name="action">
   <property name="icon">
    <iconset>
     <normaloff>../../../usr/share/icons/oxygen/48x48/actions/project-open.png</normaloff>../../../usr/share/icons/oxygen/48x48/actions/project-open.png</iconset>
   </property>
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>Помощь</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>Краткая информация</string>
   </property>
  </action>
  <action name="action_5">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Инфо</string>
   </property>
  </action>
  <action name="action_8">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/images/layer-visible-off.png</normaloff>:/images/layer-visible-off.png</iconset>
   </property>
   <property name="text">
    <string>Визуальная атака</string>
   </property>
  </action>
  <action name="action_9">
   <property name="icon">
    <iconset>
     <normaloff>:/images/edit-table-insert-row-above.png</normaloff>:/images/edit-table-insert-row-above.png</iconset>
   </property>
   <property name="text">
    <string>Формат файла</string>
   </property>
  </action>
  <action name="action_11">
   <property name="text">
    <string>Палитра</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
